spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/SnapshotTableProcedure.java

```diff
@@ -74,26 +81,21 @@ public StructType outputType() {
 
   @Override
   public InternalRow[] call(InternalRow args) {
-    String source = args.getString(0);
+
```

Comment:
unnecessary newline here

Reply:
Thank you for reviewing the code! I will improve it.